src/PrestaShopBundle/Resources/views/Admin/WebProfiler/config.html.twig

```diff
@@ -100,7 +100,7 @@
             <div class="sf-toolbar-info-piece">
                 <b>Resources</b>
                 <span>
-                    {% set appVersion = collector.applicationversion|join()[:3] %}
+                    {% set appVersion = collector.applicationversion|join()[:1] %}
```

Comment:
Because devdocs link is now https://devdocs.prestashop.com/8/ instead of https://devdocs.prestashop.com/8.0/